diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/constants/Warnings.kt

```diff
@@ -76,8 +76,8 @@ enum class Warnings(private val canBeAutoCorrected: Boolean, private val warn: S
     STRING_CONCATENATION(false, "strings should not be concatenated using plus operator - use string templates instead if the statement fits one line"),
     TOO_MANY_BLANK_LINES(true, "too many consecutive blank lines"),
     WRONG_WHITESPACE(true, "incorrect usage of whitespaces for code separation"),
+    WHEN_WITHOUT_ELSE(true, "each when statement must have else at the end"),
     TOO_MANY_CONSECUTIVE_SPACES(true, "too many consecutive spaces"),
-    ENUMS_SEPARATED(true, "split enumeration error"),
```

Comment:
Why are you so persistently removing `ENUMS_SEPARATED` in every pull request? :)